lib/build.gradle

```diff
@@ -32,17 +35,31 @@ test {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9'
     // Ktor related dependencies
     implementation "io.ktor:ktor-client-core:$ktor_version"
     implementation "io.ktor:ktor-client-android:$ktor_version"
     implementation "io.ktor:ktor-client-cio:$ktor_version"
     implementation "io.ktor:ktor-client-serialization-jvm:$ktor_version"
+    implementation "io.ktor:ktor-client-mock:$ktor_version"
     implementation "org.jetbrains.kotlinx:kotlinx-datetime:$kdatetime_version"
 
+    //Coroutines
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutines_version"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutines_version"
+
+    //Dagger
+    implementation 'com.google.dagger:dagger:2.28.3'
+    kapt 'com.google.dagger:dagger-compiler:2.28.3'
+
     implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
     implementation "org.jetbrains.kotlinx:kotlinx-serialization-json:$kserialization_version"
+
     testImplementation "io.mockk:mockk:$mockk_version"
+    testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.1.0"
+    testImplementation "org.mockito:mockito-core:3.0.0"
+    testImplementation "org.mockito:mockito-inline:3.0.0"
```

Comment:
What is Mockito required for? Can MockK potentially cover it?

Reply:
Yeah I can remove these now. I was testing with mockito but as you said MockK will cover the mocking :)